src/components/calendar/Calendar.vue

```diff
@@ -118,7 +118,8 @@
 		}
 
 		display: grid;
-		grid-template-columns: 10% repeat(v-bind('dayRangeLength'), 14%);
+		$legend-width: 10%;
+		grid-template-columns: $legend-width repeat(v-bind('dayRangeLength'), calc((100% - $legend-width) / v-bind('dayRangeLength')));
 		justify-content: center;
```

Comment:
(not your change, something I forgot to do)

Can this be put above the `&>div {}` block? And the `$legend-width` var factored to the top of this list (above `display: grid`)?

Reply:
Also nit: imo this miiiiight be worth a comment explaining how it works, but very flexible on that, is probably good as is